internal/node/controller/controller2.go

```diff
@@ -5,13 +5,16 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/bluesky-social/indigo/api/atproto"
```

Comment:
is this the one where there's `atproto` and `agnostic` that have the same functions? if so what was the difference again and can we stick to one

Reply:
Sadly it's because `atproto.RepoGetRecord` has a `util.LexTypeDecoder` and that's what I was having a tough time mocking out for tests

https://github.com/bluesky-social/indigo/blob/bfc7f487e4ed5ece2c07a1bb309ba1d32dede33e/api/atproto/repogetRecord.go

vs

https://github.com/bluesky-social/indigo/blob/bfc7f487e4ed5ece2c07a1bb309ba1d32dede33e/api/agnostic/repogetRecord.go

If there's a way to get around this though we should use `atproto`